src/main/java/com/twilio/http/TwilioRestClient.java

```diff
@@ -19,6 +25,7 @@
     private final String region;
     private final String edge;
     private final HttpClient httpClient;
+    private static final Logger logger = LogManager.getLogger("TwilioRestClient");
```

Comment:
Should `"TwilioRestClient"` be configurable?

Reply:
I don't think it would be necessary for this to be configurable because it should be the class name. I have now changed this to now use the name of the calling class based on the log4j [documentation](https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/LogManager.html) recommendation.